orttraining/orttraining/python/training/ortmodule/_custom_op_symbolic_registry.py

```diff
@@ -38,20 +41,43 @@ def pytorch_type_to_onnx(scalar_type: str) -> torch.onnx.TensorProtoDataType:
         return _CAST_PYTORCH_TO_ONNX[scalar_type]
 
 
-def wrap_custom_export_function(original_func):
-    # Starting from PyTorch 1.11, there has been a change to symbolic function signature
-    # in terms of how additional context is accessed. More info at
-    # https://github.com/pytorch/pytorch/blob/6b02648479d3615fa3260961e24f38dd0f22da94/torch/onnx/symbolic_helper.py#L48
-    # This code can be cleaned up once support for PyTorch version < 1.11 is dropped.
-    try:
+def wrap_custom_export_function(original_func: Callable) -> Callable:
+    """
+    This function is to wrap the custom export function to make sure it can be used by different versions of PyTorch.
```

Comment:
Typical usage of documentation from the [Google style guide](https://google.github.io/styleguide/pyguide.html#383-functions-and-methods) should be:

```python
"""A one-line summary of the function

An empty line followed by a more descriptive explanation of the function, example, or code samples

Args:

Returns:
"""
```

I think this is what will work with the Python API documentation tool (Sphinx) as well.